### PR TITLE
fix(client): add footer dec 2021

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Once this is merged, I will update the footer for the forum and news.

### CODE REVIEW

**Review by raisedadead — DISMISSED** [review]

File: client/i18n/locales/english/trending.json [hunk 1]
```diff
@@ -1,63 +1,65 @@
...
+  "article1title": "Linux List Processes",
+  "article2link": "https://www.freecodecamp.org/news/web-page-text-editor-how-to-open-html-code-in-mac-textedit/",
+  "article2title": "Web Page Text Editor",
+  "article3link": "https://www.freecodecamp.org/news/what-is-open-source-software-explained-in-plain-english/",
+  "article3title": "What is Open Source?",
+  "article4link": "https://www.freecodecamp.org/news/protect-yourself-against-sim-swapping-attacks/",
+  "article4title": "Sim Swapping Attacks",
+  "article5link": "https://www.freecodecamp.org/news/rng-meaning-what-does-rng-stand-for-in-gaming/",
+  "article5title": "RNG Meaning in Gaming",
+  "article6link": "https://www.freecodecamp.org/news/the-model-view-c…attern-mvc-architecture-and-frameworks-explained/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:14]:
```suggestion
  "article6link": "https://www.freecodecamp.org/news/the-model-view-controller-pattern-mvc-architecture-and-frameworks-explained/",
```

File: client/i18n/locales/english/trending.json [hunk 2]
```diff
@@ -1,63 +1,65 @@
...
+  "article6title": "Model View Controller",
+  "article7link": "https://www.freecodecamp.org/news/front-end-develo…front-end-development-explained-in-plain-english/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:16]:
```suggestion
  "article7link": "https://www.freecodecamp.org/news/front-end-developer-what-is-front-end-development-explained-in-plain-english/",
```

File: client/i18n/locales/english/trending.json [hunk 3]
```diff
@@ -1,63 +1,65 @@
...
+  "article7title": "Front End Development",
+  "article8link": "https://www.freecodecamp.org/news/what-is-a-full-s…developer-back-end-front-end-full-stack-engineer/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:18]:
```suggestion
  "article8link": "https://www.freecodecamp.org/news/what-is-a-full-stack-developer-back-end-front-end-full-stack-engineer/",
```

File: client/i18n/locales/english/trending.json [hunk 4]
```diff
@@ -1,63 +1,65 @@
...
+  "article8title": "Full Stack Developer?",
+  "article9link": "https://www.freecodecamp.org/news/javascript-switch-case-js-switch-statement-example/",
+  "article9title": "JavaScript Switch Case",
+  "article10link": "https://www.freecodecamp.org/news/bash-sleep-how-t…ke-a-shell-script-wait-n-seconds-example-command/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:22]:
```suggestion
  "article10link": "https://www.freecodecamp.org/news/bash-sleep-how-to-make-a-shell-script-wait-n-seconds-example-command/",
```

File: client/i18n/locales/english/trending.json [hunk 5]
```diff
@@ -1,63 +1,65 @@
...
+  "article15title": "HTML Line Break",
+  "article16link": "https://www.freecodecamp.org/news/how-to-convert-a-string-to-an-int-in-c-tutorial-with-example-code/",
+  "article16title": "C# String to Int",
+  "article17link": "https://www.freecodecamp.org/news/logical-fallacies-definition-fallacy-examples/",
+  "article17title": "Logical fallacies",
+  "article18link": "https://www.freecodecamp.org/news/javascript-online-html-css-js-code-editor-list-browser-ide-tools/",
+  "article18title": "JavaScript Online",
+  "article19link": "https://www.freecodecamp.org/news/sql-case-statement-tutorial-with-when-then-clause-example-queries/",
+  "article19title": "SQL Case Statement",
+  "article20link": "https://www.freecodecamp.org/news/javascript-tolow…onvert-a-string-to-lowercase-and-uppercase-in-js/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:42]:
```suggestion
  "article20link": "https://www.freecodecamp.org/news/javascript-tolowercase-how-to-convert-a-string-to-lowercase-and-uppercase-in-js/",
```

File: client/i18n/locales/english/trending.json [hunk 6]
```diff
@@ -1,63 +1,65 @@
...
+  "article20title": "JavaScript toLowerCase",
+  "article21link": "https://www.freecodecamp.org/news/angular-ngclass-example/",
+  "article21title": "Angular NgClass Example",
+  "article22link": "https://www.freecodecamp.org/news/sql-aggregate-functions-with-example-data-queries-for-beginners/",
+  "article22title": "SQL Aggregate Functions",
+  "article23link": "https://www.freecodecamp.org/news/what-is-web-development-how-to-become-a-web-developer-career-path/",
+  "article23title": "What is Web Development?",
+  "article24link": "https://www.freecodecamp.org/news/the-best-way-to-…python-python-programming-tutorial-for-beginners/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:50]:
```suggestion
  "article24link": "https://www.freecodecamp.org/news/the-best-way-to-learn-python-python-programming-tutorial-for-beginners/",
```

File: client/i18n/locales/english/trending.json [hunk 7]
```diff
@@ -1,63 +1,65 @@
...
+  "article24title": "Best Way to Learn Python",
+  "article25link": "https://www.freecodecamp.org/news/word-count-in-go…ords-and-characters-in-a-google-doc-or-word-file/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:52]:
```suggestion
  "article25link": "https://www.freecodecamp.org/news/word-count-in-google-docs-tutorial-counting-words-and-characters-in-a-google-doc-or-word-file/",
```

File: client/i18n/locales/english/trending.json [hunk 8]
```diff
@@ -1,63 +1,65 @@
...
+  "article25title": "Word Count in Google Docs",
+  "article26link": "https://www.freecodecamp.org/news/how-to-use-node-…variables-with-a-dotenv-file-for-node-js-and-npm/",
```
Comment (nhcarrigan) [inline client/i18n/locales/english/trending.json:54]:
```suggestion
  "article26link": "https://www.freecodecamp.org/news/how-to-use-node-environment-variables-with-a-dotenv-file-for-node-js-and-npm/",
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
These truncated links all 404 for me.

**Comment by ahmadabdolsaheb** [comment]
> These truncated links all 404 for me.

Thanks for catching that. 🙏

**Review by naomi-lgbt — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Pull request merged**